README.rst

```diff
@@ -227,9 +229,10 @@ Supported Devices & Software Solutions
    :term:`SMUs <SMU>`, **24xx Standard**, :term:`PI`, ✅,
    , **24xx Interactive**, :term:`TSP`, ✅, ✅
    , **26xxB**, :term:`TSP`, ✅, 🚧
+   , **2636B**, :term:`TSP`, ✅, ✅
```

Comment:
Is this separate from the 26xxB models?

Reply:
Other SMU models under 26XXB have not been fully qualified. We would need additional investigation to determine the common APIs between these listed models. Given the full qualification on available SMU2636B we would want to make SMU2636B as fully qualified instrument.